.azure-pipelines/LinuxTest.yml

```diff
@@ -105,3 +105,7 @@ jobs:
   # Ensure that any running Gradle daemon is stopped before caching starts.
   - bash: ./gradlew --stop
     displayName: Stop Gradle Daemon
+
+  - script: bash <(curl -s https://codecov.io/bash) -X gcov -f '!*/src/*/assets/*';
+    displayName: Upload to codecov.io
+    condition: eq(variables['Build.Reason'], 'Schedule')
```

Comment:
Hmm, so the upload is now split into two places. Will the statistics then be merged on the server side? If not, which of the two does the percentage in the coverage badge then refer to?

Reply:
Codecov does automatically merge multiple uploads: https://docs.codecov.io/docs/merging-reports

Reply:
Nice. Could you add that info to the commit message please?

Reply:
Done.